Exec/science/Detonation/GNUmakefile.nse_net

```diff
@@ -0,0 +1,29 @@
+PRECISION  = DOUBLE
+PROFILE    = FALSE
+
+DEBUG      = FALSE
+
+DIM        = 1
+
+COMP	   = gnu
+
+USE_MPI    = TRUE
+USE_OMP    = FALSE
+
+USE_REACT  = TRUE
+
+CASTRO_HOME = ../../..
+
+# This sets the EOS directory in $(MICROPHYSICS_HOME)/eos
+EOS_DIR     := helmholtz
+
+# This sets the EOS directory in $(MICROPHYSICS_HOME)/networks
+NETWORK_DIR := ase
+SCREEN_METHOD := chabrier1998
+
```

Comment:
I think we could merge this GNUmakefile with the main one if we do a check like:

ifeq ($(USE_NSE_NET),TRUE)
  NETWORK_DIR := ase
  SCREEN_METHOD := chabrier1998
else
  NETWORK_DIR := aprox19
endif

Reply:
then we automatically get the right stuff just be setting `USE_NSE_NET`